<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FANO</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="EncFileBut">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Encode file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DecFileBut">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Decode file</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="FileRadioButtonDec">
       <property name="text">
        <string>File</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="TextRadioButtonDec">
       <property name="text">
        <string>Text</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="FileRadioButtonEnc">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>File</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="TextRadioButtonEnc">
       <property name="text">
        <string>Text</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="TextEncIn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="TextDecOut">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Encoder</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="TextEncOut">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Decoder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EncTransf">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Transform</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DecTransf">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>290</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Transform</string>
    </property>
   </widget>
   <widget class="QLabel" name="StatusLabel1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Status: OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="StatusLabel2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Status: OK</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="TextDecIn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveFileEnc">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>210</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Save to file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveFileDec">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>460</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Save to file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearEncText">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>❌</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearDecText">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>290</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>❌</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Select an author</string>
    </property>
   </widget>
   <widget class="QComboBox" name="CBEnc">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>W/O</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tolstoy L. N.</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dostoevskiy F. M.</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pushkin A. S.</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>250</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Select an author</string>
    </property>
   </widget>
   <widget class="QComboBox" name="CBDec">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentText">
     <string>W/O</string>
    </property>
    <item>
     <property name="text">
      <string>W/O</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tolstoy L. N.</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dostoevskiy F. M.</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pushkin A. S.</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>339</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
